llama-index-core/llama_index/core/vector_stores/types.py

```diff
@@ -94,8 +123,17 @@ class MetadataFilter(BaseModel):
     """
 
     key: str
-    value: Union[StrictInt, StrictFloat, StrictStr, List[StrictStr]]
-    operator: FilterOperator = FilterOperator.EQ
+    value: Union[
+        StrictInt,
+        StrictFloat,
+        StrictStr,
+        Json,
```

Comment:
huh, what is the Json type? If it means a dict, pretty sure some vector dbs will complain if you try to insert a dict as metadata

Reply:
Milvus supports JSON typed metadata. I added this in for support there. We don't need it for our specific use case, so I can remove this if it might cause issues. I just wanted to add it to be a bit more comprehensive

Reply:
I think we should leave this as-is for now

Reply:
Reverted this